src/cli.ts

```diff
@@ -6,8 +6,13 @@ const P = new Pokedex();
 
 const fetchPokemonInfo = async (name: string) => {
   console.log(chalk.green(`Fetching information about ${name}...`));
-  const pokemonSpecies = await P.getPokemonSpeciesByName(name);
-  return pokemonSpecies;
+  try {
+    const pokemonSpecies = await P.getPokemonSpeciesByName(name);
+    return pokemonSpecies;
+  }
+  catch(error) {
```

Comment:
[style] I think this should use a similar syntax to what we use with `if / else` statements where we have the curly brace on the same line:

```
try {

} catch (error) {

}
```

Similar to:

```
if (...) {

} else {

}
```

Also, I like a space after the `catch`!